### PR TITLE
Fix typos of TupperVim

### PR DESCRIPTION
**Pull request by alexherbo2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
 
-  <a id="fork-me" href="https://github.com/tuppervim/tuppervim.github.io"><img style="position: absolute; top: 0;
+  <a id="fork-me" href="https://github.com/tupperVim/tuppervim.github.io"><img style="position: absolute; top: 0;
```
Comment (maggick) [inline index.html]: it's a link so no capital letters.

File: index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
     <a href="/"><img src="media/TupperVimlogo.svg" onerror="this.src='media/TupperVimlogo.png'" alt="TupperVim" /></a>
     <h1>TupperVim</h1>
     <p>
-      Les Tuppervim sont des évènements où l’on partage des connaissances et des
+      Les TupperVim sont des évènements où l’on partage des connaissances et des
```
Comment (maggick) [inline index.html]: ok

File: index.html [hunk 3]
```diff
@@ -150,7 +150,7 @@ <h2>Contact</h2>
         <a rel="nofollow" href="https://framalistes.org/sympa/info/tuppervim">https://framalistes.org/sympa/info/tuppervim</a>
       </li>
       <li> Twitter :
-        <a rel="nofollow" href="https://twitter.com/tuppervim">@tuppervim</a>
+        <a rel="nofollow" href="https://twitter.com/tupperVim">@tupperVim</a>
```
Comment (maggick) [inline index.html]: ok

File: index.html [hunk 4]
```diff
@@ -161,7 +161,7 @@ <h2>Contact</h2>
   <section id="archives">
     <h2>Archives</h2>
     <p><a rel="nofollow" href="https://tuppervim.org/archives.html">Une page est
-    dédiée aux archives de tuppervim.</a>
+    dédiée aux archives de TupperVim.</a>
   </section>
```
Comment (maggick) [inline index.html]: ok

File: index.html (hunk 1 above)
Comment (alexherbo2) [reply index.html]: It is the correct URL case.

**Pull request merged**